libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -326,6 +326,9 @@ export class MultiInputComponent implements
 
     /** @hidden */
     openChangeHandle(open: boolean): void {
+        if (this.disabled) {
+            return ;
+        }
         if (this.open !== open) {
```

Comment:
if else?

Reply:
???? these are 2 diff checks, I don't see the need of else block

Reply:
@stefanoScalzo This is just to break method